crates/napi/src/threadsafe_function.rs

```diff
@@ -355,8 +355,8 @@ unsafe extern "C" fn thread_finalize_cb<T: 'static, V: ToNapiValue, R>(
   R: 'static + Send + FnMut(ThreadSafeCallContext<T>) -> Result<Vec<V>>,
 {
   // cleanup
-  drop(unsafe { Box::<R>::from_raw(finalize_data.cast()) });
-  let aborted = unsafe { Arc::<Mutex<bool>>::from_raw(finalize_hint.cast()) };
+  drop(unsafe { Box::<R>::from_raw(finalize_hint.cast()) });
```

Comment:
`finalize_hint` is the `context`, see https://github.com/nodejs/node/blob/705fefd30d68e70b0b6aff07567ee0afb8324f89/src/node_api.cc#L376

Reply:
Great to hear 👍🏿